src/Tribe/Tickets.php

```diff
@@ -2921,34 +2966,50 @@ public function ticket_add( $post_id, $data ) {
 		}
 
 		/**
-		 * Get the saved or default ticket provider
+		 * Get the saved or default ticket provider.
+		 *
+		 * Will return 'Tribe__Tickets__Tickets' if there is a saved provider that is currently not active.
+		 * Example: If provider is WooCommerce Ticket but ETP is inactive, will return 'Tribe__Tickets__Tickets'.
 		 *
 		 * @since 4.7
+		 * @since TBD If detected module is inactive, return 'Tribe__Tickets__Tickets' to avoid fatals while not
+		 * 			  returning unexpected data, such as finding tickets of the wrong provider type.
 		 *
 		 * @param int $event_id - the post id of the event the ticket is attached to.
 		 *
-		 * @return string ticket module class name
+		 * @return string|false The ticket module class name (confirmed active). False if module is not active.
```

Comment:
Not returning false anymore?

Reply:
No because of always returning at least `'Tribe__Tickets__Tickets'`